src/main/resources/root.jsp

```diff
@@ -41,46 +38,281 @@
             language: '${language}',
             urlbase: '<c:url value="/modules/appshell/${appName}"/>'
         });
+        // Loader resource listener
+        const bundlesLoaded = [];
+        const loaderResourceListener = e => {
+            if (e.target.nodeName === "SCRIPT") {
+                // Count script tag in the page
+                const totalScripts = document.querySelectorAll('script').length;
+                const b = e.target.getAttribute("data-webpack")?.split(':')[0];
+
+                if (b && !bundlesLoaded.includes(b)) {
+                    bundlesLoaded.push(b);
+                    document.querySelector('.jahia-loading_text').innerHTML = '<fmt:message bundle="${bundle}" key="label.loading"/> (' + b + ' ' + bundlesLoaded.length +  ' / ${remotes.size() + 1})';
+                }
+            }
```

Comment:
May be there is an easier solution, it would be possible to attach directly a listener on the script of the remote:
`<script src="<c:url value="${remote}"/>"></script>`
May be it would be simpler than attaching a global listener to the document, I don't know may be you already tried this idea.
to me it would also allow to not have to clean for the global listener on the document if the listener is already attached to the remote script loading, will be by default executed once for it's original purpose.

Reply:
same here